### ISSUE TITLE
Impossible to count number of NFTs in large collections and get the IDs

### ISSUE THREAD

**Issue by cybercent** [onflow/cadence#2745]
### Issue to be solved

`getIDs().length` breaks when used on accounts with large NFT collections like `0x329feb3ab062d289.661249`

```
failed to retrieve slab 0x329feb3ab062d289.661249: get value failed: [Error Code: 1110] computation exceeds limit (100000)
```

### Suggested Solution

- Add way to get the count of total NFTs by avoiding `getIDs().length`
- Add way to get a slice of the IDs in the collection.

@joshuahannan I'm tagging you here as this might be interesting for the V2 standard.

**Comment by joshuahannan** [comment]
yeah I would love if we could find a solution for this that we could use with the standard

**Comment by turbolent** [comment]
These improvements are all related to the NFT standard, moving
